YAXLib/YAXLib.csproj

```diff
@@ -1,64 +1,31 @@
-        <PackageReleaseNotes>Now supporting netstandard2.0 netstandard2.1 net461 net462 net472 net48 net5.0</PackageReleaseNotes>
-        <OutputType>Library</OutputType>
-        <RootNamespace>YAXLib</RootNamespace>
-        <AssemblyName>YAXLib</AssemblyName>
-        <TargetFrameworks>netstandard2.0;netstandard2.1;net461;net462;net472;net48;net5.0</TargetFrameworks>
-        <LangVersion>8</LangVersion>
-        <WarningLevel>4</WarningLevel>
-        <Nullable>disable</Nullable>
-        <PublishUrl>publish\</PublishUrl>
-        <EnableNETAnalyzers>true</EnableNETAnalyzers>
-    </PropertyGroup>
-    <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|AnyCPU'">
-        <DefineConstants>DEBUG;TRACE</DefineConstants>
-        <OutputPath>bin\Debug\</OutputPath>
-        <DocumentationFile></DocumentationFile>
-    </PropertyGroup>
-    <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Release|AnyCPU'">
-        <OutputPath>bin\Release\</OutputPath>
-        <DocumentationFile>bin\Release\YAXLib.xml</DocumentationFile>
-        <ContinuousIntegrationBuild>true</ContinuousIntegrationBuild>
-        <Deterministic>true</Deterministic>
-        <PublishRepositoryUrl>true</PublishRepositoryUrl>
-        <EmbedUntrackedSources>true</EmbedUntrackedSources>
-        <IncludeSymbols>true</IncludeSymbols>
-        <SymbolPackageFormat>snupkg</SymbolPackageFormat>
-    </PropertyGroup>
-    <PropertyGroup Condition="'$(Configuration)|$(TargetFramework)|$(Platform)'=='Release|netstandard2.0|AnyCPU'">
-      <DefineConstants>TRACE</DefineConstants>
+        <PackageTags>XML Serialization Serializer Serialize .NET Core</PackageTags>
     </PropertyGroup>
+
+    <ItemGroup>
+        <None Include="../Logo/YAXLib_256x256.png" Pack="true" Visible="false" PackagePath="" />
+    </ItemGroup>
+
     <ItemGroup>
-        <PackageReference Include="Microsoft.DotNet.PlatformAbstractions" Version="3.1.6" />
-        <PackageReference Include="Microsoft.Extensions.DependencyModel" Version="3.1.6" />
         <PackageReference Include="Microsoft.SourceLink.GitHub" Version="1.0.0">
-          <PrivateAssets>all</PrivateAssets>
-          <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
+            <PrivateAssets>all</PrivateAssets>
+            <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
         </PackageReference>
     </ItemGroup>
+
     <ItemGroup>
         <AssemblyAttribute Include="System.Runtime.CompilerServices.InternalsVisibleToAttribute">
-            <_Parameter1>YAXLibTests, PublicKey=0024000004800000940000000602000000240000525341310004000001000100d7f87eefee568e19ff867da48567811d16853dbc045adcae7a9682012ad27a39dd24c9959db4e87965a589859f8a2cfc33c2d4997bf969dc4baa159ceefe1a85f45368d76918c764df03763848f50370e660da01bb4b2614fb67f8d0f7183c0c640b16c61d47628d8e4ba20f86dc6bab11b0528425efd05607c28ac576f92ca1</_Parameter1>

```

Comment:
have we removed strong naming?

Reply:
Isn't that because the key of the published assemblies is different from they `YAXLib.DevKey.snk`. The other key is not published in this repo. Is there an advantage of a signature for development only?

Reply:
> have we removed strong naming?

First of all wo do not have the release singing key and according to this document strong naming should normally not be necessary nor is it recommended to do so:
https://github.com/dotnet/runtime/blob/main/docs/project/strong-name-signing.md

Reply:
Well there is always a long debate on that. But I did not expect this change in this pr (scope creep?) 

Also I do it with good reasons in other open source libraries. I think microsoft is also still strong naming their binaries ?

Reply:
Agree. I reverted this.